chapter-14/recipe-01/abstract.md

```diff
@@ -0,0 +1,4 @@
+In this recipe, we will extend the test example of
+[chapter 4, recipe 1](../../chapter-04/recipe-01),
```

Comment:
What about changing the reference to the chapter to "Recipe 1 in Chapter 4"?

Reply:
I agree.